### PR TITLE
Trials Project Code Review PR

### PR DESCRIPTION
**Pull request by megha-n-bodke**
**Project Requirements**

- Created an empty repo.
- Created a new branch.
- Copied code from the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <!-- adding comments to check code from here -->
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- check html elements arrangment -->
+    <head><title></title>
```
Comment (megha-n-bodke) [inline index.html:13]: Please add a descriptive title.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="stylesheet" type="text/css" href="css/style.css"/></head>
+    <body>
+      <nav id="navbar" style="position:sticky;top:0;">
+        <a href="#welcome-section">Welcome Section</a>
+        </nav>
+      <style type="text/css">
+    @media only screen and (max-width: 600px)
```
Comment (megha-n-bodke) [inline index.html:20]: Please include CSS code in a separate file. To maintain modularity, and readability.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    {
+    #welcome-section {
+        color:#ffffff;
```
Comment (megha-n-bodke) [inline index.html:23]: Please try to give background for good looks of webpage.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    }
+      </style>
+    <!-- check this section as well for review -->
+    <section id="welcome-section">
+    <h1>Hi! I am a Software Developer</h1>
+      </section>
+      <!-- review this section as well -->
+      <section id="projects">
+      <a id="profile-link" class="project-tile" href="" target="_blank"><img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"
+    "title="project"></img>
```
Comment (megha-n-bodke) [inline index.html:34]: Please configure linters ,is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by megha-n-bodke — COMMENTED** [review]
- [ ] Remember to add README.md

**Pull request merged**